cypress/fixtures/seeds/collectionsFixture.json

```diff
@@ -193,4 +193,4 @@
     }
     
   ]
-}
+}
```

Comment:
Can we revert this please? (Just add back the new line)

Reply:
fixed